internal/cli/server/service.go

```diff
@@ -165,6 +165,10 @@ func (s *Server) ChainSetHead(ctx context.Context, req *proto.ChainSetHeadReques
 }
 
 func (s *Server) Status(ctx context.Context, _ *empty.Empty) (*proto.StatusResponse, error) {
+	if s.backend == nil {
+		return nil, fmt.Errorf("the service is currently unavailable, try again later")
```

Comment:
why fmt.Errorf ?

Reply:
Changed to `errors.New()`, thanks.

Reply:
Also possible to define a package level error variable and reuse it here.

Reply:
Done, thanks!